src/java.ts

```diff
@@ -92,11 +87,15 @@ export async function serverSupportsJREProvisioning(
   return supports;
 }
 
-async function fetchLatestSupportedJRE(serverUrl: string, platformInfo: PlatformInfo) {
+async function fetchLatestSupportedJRE(properties: ScannerProperties, platformInfo: PlatformInfo) {
+  // TODO: enforce correct mapping to SC/SQ
+  const serverUrl = properties[ScannerProperty.SonarHostUrl] ?? SONARCLOUD_PRODUCTION_URL;
```

Comment:
Once https://github.com/SonarSource/sonar-scanner-npm/pull/114 is merged, `SonarHostUrl` will already contain the final URL that should be reached

```suggestion
  const serverUrl = properties[ScannerProperty.SonarHostUrl];
```

Reply:
Perfect, I'll update this with that assumption 👍🏻